src/lib/components/helpToolTip/reactPopperTooltip.module.scss

```diff
@@ -0,0 +1,119 @@
+// copied from https://github.com/mohsinulhaq/react-popper-tooltip/blob/master/src/styles.css
+// and adopted to our styles, because just including
+// "// import 'react-popper-tooltip/dist/styles.css';" in the component doesnt work
+
+.tooltip-container {
+  background-color: white;
+  border-radius: 3px;
+  border: 1px solid silver;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.18);
+  display: flex;
+  flex-direction: column;
+  margin: 0.4rem;
+  padding: 0.4rem;
+  transition: opacity 0.3s;
+  z-index: 2147483647;
+}
+
+.tooltipArrow {
+  height: 1rem;
+  position: absolute;
+  width: 1rem;
```

Comment:
IDEM here

Reply:
Hey David! I copied these styles from the library. I tried to change these to em, but the problem is without being well-versed in CSS it's difficult to make it look the same. Also, when the library changes these styles and we copy the updated styles, we would have to do the same changes again … I know this is not ideal!

In our own styles, I changed my CSS to always use rem for fonts and em for everything else.